gitlab/main.go

```diff
@@ -0,0 +1,254 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gptscript-ai/tools/gitlab/pkg/commands"
+	gitlab "gitlab.com/gitlab-org/api/client-go"
+	"os"
+	"strconv"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Usage: gitlab <command>")
+		os.Exit(1)
+	}
+
+	command := os.Args[1]
+
+	git, err := gitlab.NewClient(os.Getenv("GITLAB_TOKEN"))
+	if err != nil {
+		fmt.Printf("Failed to create gitlab client: %v\n", err)
+		os.Exit(1)
+	}
+	user, _, err := git.Users.CurrentUser()
+	if err != nil {
+		fmt.Printf("Error getting current user: %v\n", err)
+		os.Exit(1)
+	}
+
+	switch command {
+	// Issues
+	case "queryIssues":
+		rawQuery := os.Getenv("ISSUE_QUERY")
+		var query map[string]any
+		if rawQuery != "" {
+			query, err = parseStringToMap(rawQuery)
+			if err != nil {
+				fmt.Printf("Failed to parse ISSUE_QUERY: %v\n", err)
+				os.Exit(1)
+			}
```

Comment:
nit: if you factor this switch out into a separate function, you can return errors and have a single `fmt.Println(err); os.Exit(1);` in `main()`.